allennlp_config/pico.json

```diff
@@ -0,0 +1,63 @@
+// Configuration adapted from https://gist.github.com/joelgrus/7cdb8fb2d81483a8d9ca121d9c617514
+// for BERT embeddings CRF tagger
+// Changed settings to use those specified in EBM-NLP paper
+{
+  "random_seed": std.extVar("SEED"),
+  "pytorch_seed": std.extVar("PYTORCH_SEED"),
+  "numpy_seed": std.extVar("NUMPY_SEED"),
+  "dataset_reader": {
+    "type": "ebmnlp",
+    "token_indexers": {
+      "bert": {
+        "type": "bert-pretrained",
+        "pretrained_model": std.extVar("BERT_VOCAB"),
+        "do_lowercase": std.extVar("is_lowercase"),
+        "use_starting_offsets": true
+      }
+    }
+  },
+  "train_data_path": std.extVar("TRAIN_PATH"),
+  "validation_data_path": std.extVar("DEV_PATH"),
+  "test_data_path": std.extVar("TEST_PATH"),
+  "evaluate_on_test": true,
+  "model": {
+    "type": "pico_crf_tagger",
```

Comment:
just checking, the only difference between this config and ner.jsonnet is this line, right ?

Reply:
no there are quite a few changes:
* pico has no char representations
* pico batch size is 20
* pico uses avg_f1 metric instead of f1_measure_overall metric
* pico has 2 layer LSTM instead of 1
* pico model removes a lot of the params to be passed into Reader or Model from the config

Reply:
all of this was to mimic the pico baseline model as closely as possible w/ exception of BERT embeds